proto/cosmos/crypto/keyring/types.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+package cosmos.crypto.keyring;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keyring";
+option (gogoproto.goproto_getters_all) = false;
+
+// TODO - define proto structs for all structs in cryto/kerying/info.go and remove the original structs.
+
+// Item is a keyring item stored in a keyring backend.
+message LocalInfo {
```

Comment:
We can actually do:

```proto
message KeyringEntry {
  string name = 1;
  Any pub_key = 2;

  oneof item {
    LocalInfo local = 1;
    LedgerInfo ledger  = 2;
    OfflineInfo offline  = 3;
    MultiInfo multi  = 4;
  }
}
```

so that we don't repeat `name` and `pub_key` for all Infos.

Reply:
Great idea. With this we will also avoid one level of `Any`.